councilmatic_core/management/commands/convert_rtf.py

```diff
@@ -46,39 +40,32 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-        self.connection = engine.connect()
-        self.update_all = options["update_all"]
-        self.update_empty = options["update_empty"]
-        """
-        This command converts RTF from Legistar into valid HTML.
-        The conversion employs "unoconv" - a CLI tool that imports and exports documents in LibreOffice. We run unoconv as a daemon process and kill it when the conversions finish.
-        Three steps occur: (1) querying the database for bill full_text (i.e., the RTF from Legistar),
-        (2) iteration over the query results, conversion to html, and creation of an inserts string,
-        (3) updating the full_text field with new html.
-        """
-        listener = subprocess.Popen(
-            ["unoconv", "--listener"],
-            stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
-        )
+        django_conn = connection.get_connection_params()
```

Comment:
i *think* you can just get a psycopg2 cursor by doing

``connection.cursor().connection```

and then you wouldn't need to all this.

Reply:
TIL! I wound up refactoring raw SQL out of this command entirely, so no need for a direct connection at all.